draft-ietf-moq-transport.md

```diff
@@ -1806,6 +1771,61 @@ STREAM_HEADER_PEEP {
 ~~~
 
 
+## Fetch
+
+Subscribers issues fetch requests to prior objects while the subscriptions are made to the live edge. Fetch allows objects to be delivered at a requested rate. The fetched objects are delivered on the same stream as the request.
+
+A subscriber issues a FETCH over a new bidirectional QUIC stream to a publisher to request objects from a closed range of groups within a track. Publisher responds to a FETCH request with either a FETCH_ERROR message or data corresponding to the range requested, over the same stream as the request. The object forwarding preference will not apply to fetches.
+
+The format of FETCH is as follows:
+
+~~~
+FETCH Message {
+  Track Namespace (b),
+  Track Name (b),
+  Priority (8),
+  Group Order (8),
```

Comment:
Do we really want to allow fetch in reverse group order? I guess it may be useful for some trick play, but it seems weird to get a stream with G9 O0 O1 ... G8 O0 O1 ... G7 O0 O1 ...

Reply:
I am with you on this. Let me remove it for now and see what others think,